features/networking/ovn.feature

```diff
@@ -573,3 +573,57 @@ Feature: OVN related networking scenarios
       | ping | -c | 30 | -i | 0.2 | <%= cb.pod2_ip%> |
     Then the step should succeed
     And the output should contain "0% packet loss"
+  
+  # @author anusaxen@redhat.com
+  # @case_id OCP-46285
+  @admin
+  @destructive
+  @4.10 @4.9 @4.8
+  @network-ovnkubernetes
+  @vsphere-ipi
+  @vsphere-upi
```

Comment:
just want to confirm, this can be tested on UPI, right

Reply:
Nope as apiVIP wont be there on UPI. Fixed it. Thanks